### ISSUE TITLE
cca create breaks with npm version 3.3.6

### ISSUE THREAD

**Issue by richardfriedman** [MobileChromeApps/mobile-chrome-apps#594]
When running ```cca create PROJECT_NAME``` the build fails because the underlying call to cordova is failing.   This is related to issue in cordova project - https://issues.apache.org/jira/browse/CB-9687.  Not sure of a repeatable solution, though I listed a short-term workaround below. 

```
$ cca create AppFoo
cca v0.7.4
## Creating Your Application
## Running Cordova Command: create /private/tmp/AppFoo  
cp: no such file or directory: /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/cordova-app-hello-world/hooks/*

cp: no such file or directory: /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/cordova-app-hello-world/config.xml

Parsing /private/tmp/AppFoo/config.xml failed
/Users/rfriedman/.node/lib/node_modules/cca/node_modules/q/q.js:155
                throw e;
                ^

Error: ENOENT: no such file or directory, open '/private/tmp/AppFoo/config.xml'
    at Error (native)
```worktop:tmp rfriedman$ cca create AppFoo
cca v0.7.4
## Creating Your Application
## Running Cordova Command: create /private/tmp/AppFoo  
cp: no such file or directory: /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/cordova-app-hello-world/hooks/*

cp: no such file or directory: /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/cordova-app-hello-world/config.xml

Parsing /private/tmp/AppFoo/config.xml failed
/Users/rfriedman/.node/lib/node_modules/cca/node_modules/q/q.js:155
                throw e;
                ^

Error: ENOENT: no such file or directory, open '/private/tmp/AppFoo/config.xml'
    at Error (native)
    at Object.fs.openSync (fs.js:549:18)
    at Object.fs.readFileSync (fs.js:397:15)
    at Object.module.exports.parseElementtreeSync (/Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/src/util/xml-helpers.js:118:27)
    at Object.ConfigParser (/Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/src/configparser/ConfigParser.js:33:24)
    at /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/src/cordova/create.js:252:20
    at _fulfilled (/Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/q/q.js:787:54)
    at self.promiseDispatch.done (/Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/q/q.js:816:30)
    at Promise.promise.promiseDispatch (/Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/q/q.js:749:13)
    at /Users/rfriedman/.node/lib/node_modules/cca/node_modules/cordova-lib/node_modules/q/q.js:509:49
```

The only work around I have had 
```
worktop:tmp rfriedman$ cd ~/.node/lib/node_modules/cca/node_modules/cordova-lib/
worktop:cordova-lib rfriedman$ npm install
```
After this running ```cca create AppFoo``` works.

**Comment by agrieve** [comment]
Sorry for a lack of update, but this appears to be a bug in Cordova and a fix has not yet been released. Expecting a new release some time next week.

**Comment by agrieve** [comment]
This is now (finally) fixed with cca v0.8.0!
